uvicorn/supervisors/basereload.py

```diff
@@ -86,5 +102,12 @@ def shutdown(self) -> None:
         )
         logger.info(message, extra={"color_message": color_message})
 
-    def should_restart(self) -> bool:
+    def should_restart(self) -> Optional[List[Path]]:
         raise NotImplementedError("Reload strategies should override should_restart()")
+
+
+def _display_path(path: Path) -> str:
+    try:
+        return f"'{path.relative_to(Path.cwd())}'"
+    except ValueError:
```

Comment:
I know we had this before, but do you know when do we have a `ValueError`?

Reply:
Lots! Whenever `path` is not inside `cwd`.